Tests/PSGetUpdateModuleManifest.Tests.ps1

```diff
@@ -896,4 +899,68 @@ Describe PowerShell.PSGet.UpdateModuleManifest -Tags 'BVT','InnerLoop' {
         Assert ($newModuleInfo.CompatiblePSEditions -contains $CompatiblePSEditions[1]) "CompatiblePSEditions should include $($CompatiblePSEditions[1])"
     } `
     -Skip:$($PSVersionTable.PSVersion -lt '5.1.0')
-}
\ No newline at end of file
+
+    # Purpose: Validate Update-ModuleManifest cmdlet throws warnings when passed a prerelease string with invalid characters
+    #
+    # Action:
+    #      Update-ModuleManifest -Path [Path] -PreRelease '-alpha+001'
+    #
+    # Expected Result: Update-ModuleManifest should throw InvalidCharactersInPreReleaseString errorid.
+    #
+    It UpdateModuleManifestWithInvalidPreReleaseString {
+        $PreRelease = "-alpha+001" 
+        $Version = "3.2.1"
+
+        $expectedErrorMessage = "The PreRelease string contains invalid characters. Please ensure that only characters 'a-zA-Z0-9' and possibly hyphen ('-') at the beginning are in the PreRelease string."
+        $expectedFullyQualifiedErrorId = "InvalidCharactersInPreReleaseString,Validate-PreReleaseString"
+
+        $ScriptBlock = {
+            New-ModuleManifest -path $script:testManifestPath
+            Update-ModuleManifest -Path $script:testManifestPath -PreRelease $PreRelease -ModuleVersion $Version -Confirm:$false
+        }
+
+        $ScriptBlock | Should -Throw $expectedErrorMessage -ErrorId $expectedFullyQualifiedErrorId
```

Comment:
You will have to import the localized data and use the string identifier to get the localized error message. For more details --> https://github.com/PowerShell/PowerShellGet/blob/210c879ba9beed0babecdbdd8165ebb780b5749e/Tests/PSGetUnInstallScript.Tests.ps1#L198